components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/DestinationEntity.java

```diff
@@ -0,0 +1,52 @@
+package com.sap.cloud.cmp.ord.service.storage.model;
+
+import com.sap.olingo.jpa.metadata.core.edm.annotation.EdmIgnore;
+import com.sap.olingo.jpa.metadata.core.edm.annotation.EdmProtectedBy;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import org.eclipse.persistence.annotations.Convert;
+import org.eclipse.persistence.annotations.TypeConverter;
+
+import java.util.Set;
+import java.util.UUID;
+
+@Entity(name = "destination")
+@Table(name = "tenants_destinations")
+public class DestinationEntity {
+    @javax.persistence.Id
+    @Column(name = "id")
+    @Convert("uuidConverter")
+    @TypeConverter(name = "uuidConverter", dataType = Object.class, objectType = UUID.class)
+    private UUID Id;
+
+    @Column(name = "name", length = Integer.MAX_VALUE)
+    private String name;
+
+    @Column(name = "type", length = Integer.MAX_VALUE)
+    private String type;
+
+    @Column(name = "url", length = Integer.MAX_VALUE)
+    private String url;
+
+    @EdmProtectedBy(name = "tenant_id")
+    @EdmIgnore
+    @Column(name = "tenant_id", length = 256)
+    private String tenant;
```

Comment:
Convert the `tenant` from String to UUID. Reference - https://github.com/kyma-incubator/ord-service/pull/78